src/app/api/gql/fragment-dataset-metadata-summary.graphql

```diff
@@ -37,7 +29,24 @@ fragment DatasetMetadataDetails on DatasetMetadata {
       }
     }
   }
-  currentLicense {
-    ...License
+  currentSchema(format: PARQUET_JSON) {
+    format
+    content
+    __typename
+  }
+  __typename
+  # Last update
+  chain {
+    blocks(page: 0, perPage: 1) {
+      nodes {
+        ...MetadataBlock
+      }
+      totalCount
+      pageInfo {
```

Comment:
I think we can extract pagination info fragment. It is used here and in search requests.

Reply:
Move to separate fragment